lib/kitchen/directions/bake_inline_lists.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Kitchen
+  module Directions
+    # Bakes inline lists with the desired list separator
+    # Does not separate the last list item
+    #
+    module BakeInlineLists
+      def self.v1(book:)
+        list_separator = '; '
+        separator_class = '-os-inline-list-separator'
+
+        inline_lists = book.search('span[data-display="inline"][data-type="list"]')
```

Comment:
Is doing the entire book at once okay? It doesn't need any chapter data, and doesn't rely on/break any order of operations of the rest of the recipe.

Reply:
This might've just slowed down the recipe by a lot. New plan